### ISSUE TITLE
console is updated depending on timescale

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#1655]
Related to #1553.

The console UI is scaled with timescale, and I think it makes no sense.

**Comment by necessarily-equal** [comment]
Agreed. I feel like whatever solution is found for https://github.com/Unvanquished/Unvanquished/pull/1553 should be applied here too.
